src/envo/envo-edit.owl

```diff
@@ -40112,6 +40115,40 @@ During curing process, well-defined and irreversible chemical networks form in t
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105005> "thermoset polymer"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105005> <http://purl.obolibrary.org/obo/CHEBI_60027>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105006> (piece of plastic foam)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105006> "A piece of plastic which 1) is partially elastic so that it can deform readily under pressure and 2) has a near-spherical or granular shape."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105006> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105006> <https://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105006> <http://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105006> "2021-05-04T13:42:05Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105006> "Plastikschaumstück"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105006> "Schaumstoffstück"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym> <http://purl.obolibrary.org/obo/ENVO_06105006> "expanded polystyrene (EPS)"@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym> <http://purl.obolibrary.org/obo/ENVO_06105006> "polyurethane (PUR)"@en)
```

Comment:
I believe "polyurethane" ([xRef](https://en.wikipedia.org/wiki/Polyurethane)) refers to the polymer which is used in the manufacture of certain foams, not the piece of plastic foam. 

Consider saying "piece of polyurethane (PUR) foam" instead.